### PR TITLE
Adding CVE-2022-22965

### PR DESCRIPTION
**Pull request by ShanmukhaSahiti**

### CODE REVIEW

File: CVE-2022-22965/ubuntu1.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        - docker-buildx
+
+    - name: Create /opt/spring4shell
+      file:
+        path: /opt/spring4shell
+        state: directory
+
+    - name: Clone spring4shell
+      git:
+        repo: https://github.com/lunasec-io/Spring4Shell-POC.git
```
Comment (olegbck) [inline CVE-2022-22965/ubuntu1.yml:27]: Please add the files from https://github.com/lunasec-io/Spring4Shell-POC to the data folder, do not pull them by URL

File: CVE-2022-22965/cvex.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+blueprint: ubuntu2204-ubuntu2204
+ubuntu1:
+  playbook: ubuntu1.yml
+ubuntu2:
+  playbook: ubuntu2.yml
+  trace: "curl"
+  command:
+    - "python3 /opt/spring4shell/exploit.py --url \"http://%ubuntu1%:8080/helloworld/greeting\""
+    - "sleep 30"
```
Comment (olegbck) [inline CVE-2022-22965/cvex.yml:9]: Is the sleep really necessary? Why?

Comment (ShanmukhaSahiti) [reply CVE-2022-22965/cvex.yml:9]: The vulnerable app takes a while to start. Can only curl after it starts

File: CVE-2022-22965/ubuntu1.yml (hunk 1 above)
Comment (ShanmukhaSahiti) [reply CVE-2022-22965/ubuntu1.yml:27]: Ok. I will update that. Thank you!

Comment (ShanmukhaSahiti) [reply CVE-2022-22965/ubuntu1.yml]: Added the app files to data. Please review

**Comment by olegbck** [comment]
Great, it works. Just one more thing before I merge it: please remove .DS_Store files. We don't need them, right?

**Comment by ShanmukhaSahiti** [comment]
Right. Deleted the .DS_Store files.
Thank you!

**Pull request merged**